extensions/sql-migration/src/wizard/wizardController.ts

```diff
@@ -106,6 +114,10 @@ export class WizardController {
 		saveAndCloseButton.onClick(async () => {
```

Comment:
please wrap onClick() with
this._disposables.push(

Reply:
Addressing in another PR https://github.com/microsoft/azuredatastudio/pull/18345